hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/RDBSstFileWriter.java

```diff
@@ -82,19 +82,23 @@ public void close() throws IOException {
         throw toIOException("Failed to finish dumping into file "
             + sstFile.getAbsolutePath(), e);
       } finally {
-        sstFileWriter.close();
-        sstFileWriter = null;
-        emptyOption.close();
+        closeResources();
       }
 
       keyCounter.set(0);
     }
   }
 
+  private void closeResources() {
+    sstFileWriter.close();
```

Comment:
Check if not null.

Reply:
Null check is done before every usage of this method.